src/server/database/models/index.js

```diff
@@ -20,10 +20,13 @@ let db = {}
 //Use the DATABASE_URL String in the environment to connect to the database if it exists
 if (process.env['DATABASE_URL']) {
   var options = {}
+  console.log("Before if test environment")
 
   //Disable logging while testing
   if (process.env.NODE_ENV === 'test') {
-    options = {logging: false}
```

Comment:
Dette skal være her for at alle andre ikke skal se loggen under testing, det ble innført for å ha oversikt over resultatene til testene.